tests/nodejs/migrations.test.ts

```diff
@@ -1,5 +1,6 @@
+import type {FsMigrations} from 'knex/lib/migrations/migrate/sources/fs-migrations.js'
 import {join} from 'node:path'
-import {KyselyFsMigrationSource} from '../..'
+import {KyselyFsMigrationSource} from '../../dist/esm/migrations'
```

Comment:
personally more used to testing `src` rather than `dist` code, so not sure what the best practice is for this kind of import. (was also confused how `../..` imports were working here to begin with. 😅)

 `../../migrations` doesnt seem to work (looks like at this stage the package `exports` definitions are not being honored)

I _think_ `../..` technically imports from the `"main"` entry, which maps to the `cjs` dist code. if so, would be more inline to impot from `cjs` here, but felt also weird to explicitly import (legacy?) CJS in ESM.

alternatively, could import from `src`, but this would alter the testing strategy here.

Reply:
I think the nested import is fine.

Testing `dist` is the ultimate guarantee we ship the spec and didn't fuck up somewhere. The "production" build process being really fast makes this viable.

Reply:
makes sense! will leaves as-is, using the `esm` import for now. mocha/nodejs seems fine with this, even _if_ `../..` resolves to the CJS build